updating readme instructions for postgres

beaesguerra opened the pull request.

Commit:
updating readme instructions for postgres

ExtraGravity commented on README.adoc:
```diff
@@ -145,13 +145,33 @@ Custom tags can be added to the tests for filtering in the future.
+. Using the user you just created, create a database and grant the proper permissions to the user. The recommended default name for the database is `agricoredev`.
+
+	psql -U boresha
+	CREATE DATABASE agricoredev;
```
Should be `agricore_dev` to match the example.

beaesguerra replied:
fixed

ExtraGravity commented on README.adoc:
```diff
@@ -145,13 +145,33 @@ Custom tags can be added to the tests for filtering in the future.
-. Run the initialization commands for creating the database tables, `\i db-init.sql`, the shell must be started in the same place as `db-init.sql` for this step.
+. Create a user with correct permissions to create a database. The recommended default username is `boresha`.
+
+	CREATE ROLE boresha WITH LOGIN PASSWORD '<pw>';
```
You can put these in code blocks.

```asciidoc
[source,postgres]
----
CREATE ROLE...
----
```

beaesguerra replied:
fixed

Commit:
update readme

Commit:
Merge branch 'master' into updateReadme

ExtraGravity committed:
Fix database readme numbering

Adding the ---- sections as is created new sections and broke the numbering.
You just need a + to get add the source code sections as part of the numbers.
Sorry Bea, I didn't feel like explaining it!
